src/main/java/frc/robot/commands/dropping/AutoAlignCommand.java

```diff
@@ -0,0 +1,239 @@
+            case B3 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.C1_HYBRID, PlacePosition.C1_MID, PlacePosition.C1_HIGH);
+            case C1 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.C2_HYBRID, PlacePosition.C2_MID, PlacePosition.C2_HIGH);
+            case C2 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.C3_HYBRID, PlacePosition.C3_MID, PlacePosition.C3_HIGH);
+            default -> throw new RuntimeException("Position not found!");
+        });
+    }
+
+    /**
+     * Aligns to the node immediately right of the currently selected node (from the perspective of the drivers).
+     * This is a no-op if there isn't yet a node selected, or if the selected node is the right-most node already (A1).
+     */
+    public void alignRight() {
+        if (targetPlacePosition == null) return;
+        if (targetPlacePosition.placePosition == FieldPosition.A1) return;
+
+        // TODO: better way of doing this?
+        ElevatorState currentElevatorState = tiltedElevatorSubsystem.getState();
+        scheduleAlignCommandWith(switch (targetPlacePosition.placePosition) {
+            case A2 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.A1_HYBRID, PlacePosition.A1_MID, PlacePosition.A1_HIGH);
+            case A3 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.A2_HYBRID, PlacePosition.A2_MID, PlacePosition.A2_HIGH);
+            case B1 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.A3_HYBRID, PlacePosition.A3_MID, PlacePosition.A3_HIGH);
+            case B2 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.B1_HYBRID, PlacePosition.B1_MID, PlacePosition.B1_HIGH);
+            case B3 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.B2_HYBRID, PlacePosition.B2_MID, PlacePosition.B2_HIGH);
+            case C1 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.B3_HYBRID, PlacePosition.B3_MID, PlacePosition.B3_HIGH);
+            case C2 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.C1_HYBRID, PlacePosition.C1_MID, PlacePosition.C1_HIGH);
+            case C3 -> elevatorStateToPlacePosition(currentElevatorState, PlacePosition.C2_HYBRID, PlacePosition.C2_MID, PlacePosition.C2_HIGH);
+            default -> throw new RuntimeException("Position not found!");
+        });
+    }
+
+    /**
+     * Returns a provided HYBRID, MID, or HIGH place position, depending on the given elevator state.
+     * @param elevatorState The current elevator state of the robot.
+     * @param hybridPosition The HYBRID position to return.
+     * @param midPosition The MID position to return.
+     * @param highPosition The HIGH position to return.
+     * @return The provided HYBRID, MID, or HIGH place position, depending on the given elevator state.
+     */
+    private static PlacePosition elevatorStateToPlacePosition(
```

Comment:
this doesn't seem to be the most clean solution, but the alternative might be a big change. i'm thinking a class containing two enums. this would allow us to make methods that return a new instance of the class with one of the two enums different -- essentially, decouple the two states we're trying to decouple